resources/views/Item/Components/ItemImageCarousel.twig

```diff
@@ -1,22 +1,23 @@
+
+
 <script type="x/template" id="vue-item-image-carousel">
     <div itemscope itemtype="http://schema.org/Thing">
         <div id="single-carousel" class="owl-carousel owl-theme owl-single-item" ref="single">
-            <div v-for="image in singleImages">
+            <div v-for="image in singleImages.slice(0, quantity)">
```

Comment:
`singleImages` ist schon ein dynamischer Getter. Das begrenzen auf die eingestellte maximalanzahl ist also besser in dem Getter aufgehoben:

```
singleImages()
{
    return this.orderByPosition(
        this.$options.filters.itemImages(
            this.currentVariation.documents[0].data.images, 
            this.imageUrlAccessor
        )
    ).slice(0, this.quantity);
},
```

Reply:
done